contracts/external/cw-token-swap/src/contract.rs

```diff
@@ -217,7 +222,8 @@ pub fn execute_withdraw(deps: DepsMut, info: MessageInfo) -> Result<Response, Co
     let message = counterparty
         .promise
         .clone()
-        .into_send_message(&counterparty.address)?;
+        .into_send_message(counterparty.address.to_string(), true)?[0]
+        .clone();
```

Comment:
why do we need to take the first index and clone? should the signature be `into_send_messages`?